backend/package.json

```
@@ -18,7 +19,7 @@
     "lint": "npm run lint:js && npm run lint:type",
     "start:dev": "nodemon --ignore ./tests --exec ts-node --files -r tsconfig-paths/register ./src/server.ts",
     "prestart": "npm run migrate",
-    "start": "node ./src/server.js",
+    "start": "node ./build/src/server.js",
```

```suggestion
    "start": "node ./src/server.js",
```
the deployment will be broken. return it back